.eslintrc.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   root: true,
   extends: '@dooboo/eslint-config',
+  rules: {
+    'jsx-quotes': ['error', 'prefer-double'],
```

Comment:
Are you sure we need this? This is already defined in `@dooboo/eslint-config`.

Reply:
really? `jsx-quotes` errors happened in circleci test. so after I added this rules, errors disappeared.

Reply:
This is strange.. It is already defined in our rule [here](https://github.com/dooboolab/eslint-config/blob/8b62bb9ec7dcb83a1f2c5310aa41d600f42ff528/index.js#L35).

Reply:
I removed rules again in the commit of [2dca9bf](https://github.com/dooboolab/hackatalk-mobile/pull/23/commits/2dca9bff486577b172d48ad43c23072ce051c59c). and it passed... it's strange..